packages/dropdowns/stories/examples/Multiselect/Default.tsx

```diff
@@ -0,0 +1,231 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { useRef, useState, useEffect } from 'react';
+import debounce from 'lodash.debounce';
+import { Story } from '@storybook/react';
+import { Tag } from '@zendeskgarden/react-tags';
+import { Col, Grid, Row } from '@zendeskgarden/react-grid';
+import {
+  Item,
+  Hint,
+  Menu,
+  Label,
+  Field,
+  Dropdown,
+  Multiselect,
+  Message
+} from '@zendeskgarden/react-dropdowns';
+import { VALIDATION } from '../../../src/utils/validation';
+import StarStrokeIcon from '@zendeskgarden/svg-icons/src/12/star-stroke.svg';
+
+interface IStoryProps {
+  placement:
+    | 'auto'
+    | 'top'
+    | 'top-start'
+    | 'top-end'
+    | 'bottom'
+    | 'bottom-start'
+    | 'bottom-end'
+    | 'end'
+    | 'end-top'
+    | 'end-bottom'
+    | 'start'
+    | 'start-top'
+    | 'start-bottom';
+  isAnimated: boolean;
+  hasArrow: boolean;
+  isCompact: boolean;
+  isBare: boolean;
+  disabled: boolean;
+  focusInset: boolean;
+  placeholder: string;
+  validation: VALIDATION;
+  maxItems: number;
+  showMessage: boolean;
+  showStartIcon: boolean;
+}
+
+const options = [
+  'Aster',
+  "Bachelor's button",
+  'Celosia',
+  'Dusty miller',
+  'Everlasting winged',
+  "Four o'clock",
+  'Geranium',
+  'Honesty',
+  'Impatiens',
+  'Johnny jump-up',
+  'Kale',
+  'Lobelia',
+  'Marigold',
+  'Nasturtium',
+  'Ocimum (basil)',
+  'Petunia',
+  'Quaking grass',
+  'Rose moss',
+  'Salvia',
+  'Torenia',
+  'Ursinia',
+  'Verbena',
+  'Wax begonia',
+  'Xeranthemum',
+  'Yellow cosmos',
+  'Zinnia'
+];
+
+export const Default: Story<IStoryProps> = ({
+  isAnimated,
```

Comment:
`isAnimated` is not a valid prop for `Multiselect`. See https://garden.zendesk.com/components/multiselect#multiselect